test/metabase/lib/test_util/metadata_providers/remap.cljc

```diff
@@ -3,17 +3,18 @@
    [metabase.lib.dispatch :as lib.dispatch]
    [metabase.lib.metadata :as lib.metadata]
    [metabase.lib.schema.id :as lib.schema.id]
+   [metabase.lib.schema.metadata :as lib.schema.metadata]
    [metabase.lib.test-util.metadata-providers.mock
     :as lib.tu.metadata-providers.mock]
    [metabase.lib.util :as lib.util]
    [metabase.util :as u]
    [metabase.util.malli :as mu]))
 
-(mu/defn ^:private external-remapped-column :- lib.metadata/ColumnMetadata
+(mu/defn ^:private external-remapped-column :- ::lib.schema.metadata/column
```

Comment:
preferrable to use the new `lib.schema` keywords, we don't need the old ones like `lib.metadata/ColumnMetadata` so we can hopefully get rid of them soon